### PR TITLE
Parser improvements

### PR DESCRIPTION
**Pull request by liealex**

### CODE REVIEW

File: examples/data_collect/main.cpp
Comment (dNechita) [inline examples/data_collect/main.cpp:1]: Please undo all this formatting. It's quite time consuming to search for you changes withing all these spaces and tabs changes.

File: dependencies/adi/command_parser/CMakeLists.txt [hunk 1]
```diff
@@ -5,6 +5,19 @@ add_library(${PROJECT_NAME} STATIC
     command_parser.cpp
 )
 
+if(USE_GLOG)
```
Comment (dNechita) [inline dependencies/adi/command_parser/CMakeLists.txt:8]: I would not make command_parser depend of glog. You should stick to have the parser return error codes and let client code to print error messages or help menu.

File: examples/data_collect/main.cpp
Comment (liealex) [reply examples/data_collect/main.cpp:1]: Formatting undone

File: dependencies/adi/command_parser/CMakeLists.txt (hunk 1 above)
Comment (liealex) [reply dependencies/adi/command_parser/CMakeLists.txt]: Parser is no loger depended in glog. Kept the LOGs commented to further help me until they are no longer needed

File: dependencies/adi/command_parser/command_parser.cpp [hunk 2]
```diff
@@ -24,15 +24,11 @@
 
 #include "command_parser.h"
 #include <iostream>
+#include <map>
```
Comment (dNechita) [inline dependencies/adi/command_parser/command_parser.cpp:27]: You already included <map> in command_parser.h

Comment (liealex) [reply dependencies/adi/command_parser/command_parser.cpp]: Solved

**Review by dNechita — APPROVED** [review]

**Pull request merged**